zerver/models/prereg_users.py

```diff
@@ -73,6 +73,7 @@ class PreregistrationUser(models.Model):
     full_name = models.CharField(max_length=UserProfile.MAX_NAME_LENGTH, null=True)
     full_name_validated = models.BooleanField(default=False)
     referred_by = models.ForeignKey(UserProfile, null=True, on_delete=CASCADE)
+    send_notification = models.BooleanField(default=True, null=True)
```

Comment:
I am not sure why `null=True` is needed here. The value would be set as `True` anyways.

I am not completely sure what should be the correct default value here but we can keep it as it is for now. We can probably have a more verbose name here like `send_acceptance_notification_to_referrer` but I am not sure, can be changed later after feedback from Tim.